### ISSUE TITLE
Editing a smart object does not trigger a regeneration of the group image asset

### ISSUE THREAD

**Issue by timriot** [kilianc/adobe-generator#79]
I have confirmed with pub sub that an event is sent when this happens.

**Comment by kilianc** [comment]
Not sure...

There is a save event, being fired when you commit the smart object but, no documentId is passed :-1: 

Do we really want do dump every time a user changes a smart object without the actual "commit"?
That would mean inconsistency between ps and ML.

**Comment by timriot** [comment]
I agree with you, that if you're in a smart object, but have not commited the change, nothing should be dumped.

-Tim
